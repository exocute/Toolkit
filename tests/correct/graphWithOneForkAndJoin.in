	// should work

Graph graph
	Import imp1
	Export exp2
	Activity act1 act.ex
	Activity act2 act.ex.other
	Activity act3 act.ex.other.other
	Activity act4 act.ex.other.something
	connection act1->act3,act2
	connection act3->act4
	connection act2->act4
