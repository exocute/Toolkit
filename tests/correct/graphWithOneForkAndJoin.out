Name: graph
ImportName: imp1
ExportName: exp2
Id: act1, Name: act.ex
Id: act4, Name: act.ex.other.something
Id: act3, Name: act.ex.other.other
Id: act2, Name: act.ex.other
act1:act2,act3
act3:act4
act2:act4